CPT331.iOS/CPT331.iOS/EventSubcategory.swift

```diff
@@ -0,0 +1,468 @@
+//
+//  EventSubcategory.swift
+//  CPT331.iOS
+//
+//  Created by Peter Weller on 8/10/2016.
+//  Copyright © 2016 Peter Weller. All rights reserved.
+//
+
+import UIKit
+
+enum EventSubcategry {
+    
```

Comment:
Again could be a class? 
Actually, it could use the same class as EventCategory.
``` 
// in c# because I'm not completely familiar with swing syntax
class EventCategory
{
  private string _name
  private List<EventCategory> _subCategories;

  public EventCategory(string name, List<EventCategory> subCategories)
  {
     _name = name;
     _subCategories = subCategories;
  }
  public string Name
  {
    get { return _name; }
  }
  public List<EventCategory> ChildCategories
  {
    get { return _childCategories; }
  }
}
```